packages/golden_toolkit/lib/src/golden_builder.dart

```diff
@@ -208,6 +209,7 @@ Future<void> screenMatchesGolden(
   return expectLater(
     actualFinder,
     matchesGoldenFile(fileName),
+    skip: skip,
```

Comment:
I'm missing some context for this. Do we have a use case where we want to skip this assertion but not the entire test?

Reply:
If your development platform is not the same as your CI platform, you might have to use it.